BetterHolidays/markets/holidays.py

```
@@ -0,0 +1,159 @@
+        day = self.get_date(year)
+        if day.weekday() not in self.days:
+            if day.weekday() in self.change:
+                day += dt.timedelta(days=self.change[day.weekday()])
+
+        if self.start and day < self.start:
+            return None
+      
+        if self.end and day > self.end:
+            return None
+        
+        if type(self.type) == Holiday:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)